### PR TITLE
CLI cloudevents.md

### PR DESCRIPTION
**Pull request by Kirdock**
Current `cloudevents.md` generated with the CLI
Signed-off-by: Klaus Strießnig <k.striessnig@gmail.com>

### CODE REVIEW

File: cloudevents.md [hunk 1]
```diff
@@ -10,193 +11,15 @@
 * [Remediation](#remediation)
 * [Action](#action)
 * [Get-SLI](#get-sli)
+* [Monitoring](#monitoring)
 * [Problem](#problem)
----
-
-## Keptn CloudEvents
```
Comment (johannes-b) [inline cloudevents.md:16]: Pleaase do not remove the section ## Keptn CloudEvents

File: cloudevents.md [hunk 2]
```diff
@@ -707,61 +527,34 @@ sh.keptn.event.service.create.finished
...
   "source": "source-service",
   "specversion": "1.0",
+  "triggeredid": "3f9640b6-1d2a-4f11-95f5-23259f1d82d6",
   "type": "sh.keptn.event.service.create.finished"
 }
 ```
 
 ([&uarr; up to index](#keptn-cloud-events))
-## Approval
-### Approval Triggered
```
Comment (johannes-b) [inline cloudevents.md:725]: Why do we lose the Approval Triggered?

File: cloudevents.md [hunk 3]
```diff
@@ -1509,134 +1270,20 @@ sh.keptn.event.deployment.finished
...
+  "shkeptncontext": "a3e5f16d-8888-4720-82c7-6995062905c1",
   "source": "source-service",
   "specversion": "1.0",
+  "triggeredid": "3f9640b6-1d2a-4f11-95f5-23259f1d82d6",
   "type": "sh.keptn.event.deployment.finished"
 }
 ```
 
 ([&uarr; up to index](#keptn-cloud-events))
-## Test
```
Comment (johannes-b) [inline cloudevents.md:1519]: Why do we lose the Test Triggered?

File: cloudevents.md [hunk 4]
```diff
@@ -3638,171 +3432,34 @@ sh.keptn.event.action.finished
...
+  "shkeptncontext": "a3e5f16d-8888-4720-82c7-6995062905c1",
   "source": "source-service",
   "specversion": "1.0",
+  "triggeredid": "3f9640b6-1d2a-4f11-95f5-23259f1d82d6",
   "type": "sh.keptn.event.action.finished"
 }
 ```
 
 ([&uarr; up to index](#keptn-cloud-events))
-## Get SLI
```
Comment (johannes-b) [inline cloudevents.md:3666]: Why do we lose Get SLI Triggered?

**Review by bacherfl — APPROVED** [review]
lgtm

**Pull request merged**